mw-webapp/src/logic/allUsersPage/AllUsersPage.tsx

```diff
@@ -132,12 +132,26 @@ export const AllUsersPage = () => {
             onChange={setEmail}
             placeholder={LanguageService.allUsers.filterBlock.emailPlaceholder[language]}
             className={styles.inputFilter}
+            componentIcon={
+              <Icon
+                size={IconSize.SMALL}
+                name="SearchIcon"
+                className={styles.inputIcon}
+              />
+            }
           />
           <Input
             value={name}
             onChange={setName}
             placeholder={LanguageService.allUsers.filterBlock.namePlaceholder[language]}
             className={styles.inputFilter}
+            componentIcon={
```

Comment:
too wide interface (described in the vidoe)

Reply:
fixed